packages/kotti-ui/source/kotti-line/KtLine.vue

```diff
@@ -33,15 +56,21 @@ export default defineComponent<KottiLine.InternalProps>({
 		flex: 1 1 auto;
 		width: 100%;
 		height: 1px;
-		background: var(--gray-30);
+		background: var(--ui-02);
 	}
 
 	&__text {
 		flex: 1 0 auto;
 		padding: 0 0.4rem;
 		font-size: 0.6rem;
-		color: var(--gray-30);
+		color: var(--text-03);
 		text-align: center;
+
+		&--is-interactive {
+			font-size: 0.8rem;
```

Comment:
why the interactive `font-size` bigger


if the purpose was to make it bigger during `hover` then you're missing `:hover` for example, but I'd refrain from this and just have the `interactive-hover` token (i can't remember the exact token name but we have one for `hover` and one for `active` i.e. when you r clicking the link)

Reply:
It's actually like this in the design https://www.figma.com/file/0yFVivSWXgFf2ddEF92zkf/Kotti-Design-System?node-id=95%3A771

Reply:
I don't know why it be like that. It makes sense that the interactive ones are big, because they are supposed to be clickable. So I guess they wanted to have that, without futzing with the old styles because they normal title is not supposed to draw your attention like that.

Reply:
Can you double check with Designers that this is intentional

I'm not sure if this was design-driven or if you found it out because I am not aware of a Yoda issue that matches

more often than not, you need to get back to the designers who don't intentionally add everything

Also - the text is still not clickable according to the code changes